crates/feldera-storage/src/backend/fs.rs

```diff
@@ -0,0 +1,56 @@
+use crate::backend::StorageBackend;
+use rkyv::AlignedVec;
+use std::collections::VecDeque;
+use std::fs::File;
+use std::io;
+use std::os::unix::fs::FileExt;
+
+pub(crate) struct PosixBackend {
```

Comment:
It might not be worth doing a detailed review of the storage code (in the `backend` directory) now, because it's not what I'm planning to merge (it will be merged, revised and in different form, by @gz). I should have said that in the PR description; sorry about that.